webapi/src/main/scala/org/knora/webapi/responders/v1/GroupsResponderV1.scala

```diff
@@ -19,10 +19,10 @@ package org.knora.webapi.responders.v1
 import java.util.UUID
 
 import akka.actor.Status
+import akka.http.scaladsl.util.FastFuture
```

Comment:
Please add something about `FastFuture` to `docs/rst/knora-api-server/design-documentation/futures-with-akka.rst`, at least a link to docs elsewhere.